### ISSUE TITLE
False negatives for semicolons followed by two comments in no-extra-semicolons

### ISSUE THREAD

**Issue by Bigdragon13th** [stylelint/stylelint#2678]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`"no-extra-semicolons": true` is not error if the issue code is followed by 2 or more lines of comment. It's error if only followed by 1 comment line or none.

> Which rule, if any, is this issue related to?

`no-extra-semicolons`

> What CSS is needed to reproduce this issue?

This code not throw error.

```less
.foo {
	color: red;;
	//
	//
}
```

This one does.

```less
.foo {
	color: red;;
	//
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-extra-semicolons": true
  }
}
```

> Which version of stylelint are you using?

`7.11.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

I'm using it with gulp. CLI is having the same issue.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

LESS files.

> What did you expect to happen?

It should throw an error.

> What actually happened (e.g. what warnings or errors you are getting)?

It's not.

**Comment by jeddy3** [comment]
@Bigdragon13th Thanks for creating the issue and for using the template.

I can reproduce this with standard syntax too.

The following _will_ warn:

```css
a {
  color: #FFF;;
  /* test */
}
```

Whereas the following _will not_:

```css
a {
  color: #FFF;;
  /* test */
  /* test */
}
```

Labelling as a bug and help wanted. @Bigdragon13th Feel free to [contribute a fix](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule) if you want to help out.

**Comment by ntwb** [comment]
Fixed via #2814
